catalog/app/containers/Bucket/File.js

```diff
@@ -401,8 +402,13 @@ export default function File({
 
   const handle = { bucket, key: path, version }
 
+  const editorState = FileEditor.useState(handle)
+
   const previewOptions = React.useMemo(
-    () => ({ context: Preview.CONTEXT.FILE, mode: viewModes.mode }),
+    () => ({
```

Comment:
seems like this change is accidental (or related to updated prettier logic?)

Reply:
I added one more key/value, then removed it but the formatting persisted :man_shrugging: